fetch.bs

```diff
@@ -5423,20 +5409,28 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
          <li><p>If <var>bytes</var> is failure, then <a lt=terminated for=fetch>terminate</a> the
          ongoing fetch.
 
-         <li><p><a for=ReadableStream>Enqueue</a> a {{Uint8Array}} wrapping an {{ArrayBuffer}}
-         containing <var>bytes</var> into <var>stream</var>.
+         <li>
+          <p>Let <var>handleResponseBytes</var> be these steps:
 
-         <li><p>If <var>stream</var> is <a for=ReadableStream>errored</a>, then
-         <a lt=terminated for=fetch>terminate</a> the ongoing fetch.
+          <ol>
+           <li><p><a for=ReadableStream>Enqueue</a> a {{Uint8Array}} wrapping an {{ArrayBuffer}}
+           containing <var>bytes</var> into <var>stream</var>.
```

Comment:
FWIW this can be made rigorous these days with https://heycam.github.io/webidl/#arraybufferview-create (`[=ArrayBufferView/new=] {{Uint8Array}} given |bytes| in some realm`).